recipe/meta.yaml

```
@@ -59,6 +65,7 @@ test:
     - {{ compiler('c') }}  # [linux or win]
     - cython >=3.0
     - setuptools >=67.0
+    - typing-extensions
```

```suggestion
    - typing-extensions >=4.9
```
https://github.com/sphinx-doc/sphinx/blob/v8.2.3/pyproject.toml#L110-L116